cswinrt/code_writers.h

```diff
@@ -3410,6 +3448,17 @@ remove => %.Unsubscribe(value);
                 {
                     format_string = "(char)%";
                 }
+                else if (category == param_category::ref)
+                {
+                    if (!use_pointers)
+                    {
+                        format_string = "in __%";
+                    }
+                    else
+                    {
+                        format_string = use_pointers ? "*%" : "in %";
```

Comment:
use_pointers will never be false due to this is in the else of if !use_pointers.  And interestingly we set a different format_string between both?

Reply:
ah, thanks - missed that refactoring.  yes, with pointers, we can just deref and pass the param directly.  but C# doesn't permit in/ref/out in lambdas, so we have to capture (and unfortunately, lose the perf gain of pass by ref for netstandard)